README.md

```diff
@@ -91,11 +91,14 @@ Simply decrypt the file, add the information and encrypt the file again, using t
 
 See parameter key convention names defined on [the handbook](https://handbook.dev.codacy.org/engineering/guidelines/application-parameters.html#ssm-parameter-conventions).
 
+>All values must be enclosed in `''`
```

Comment:
Do double quotes work as well?

Reply:
Good point, I've tested and yes, it works.

Edited the commit as well